Tasks/SonarQubePostTest/PRCA/PostComments-Module.psm1

```diff
@@ -426,6 +426,9 @@ function CreateDiscussionThreads
         # add a custom property to be able to distinguish all comments created this way        
         $properties.Add($PostCommentsModule_CommentSourcePropertyName, $script:messageSource)
         
+        # A VSTS UI extension will recognize this and format the comments differently
+        $properties.Add("CodeAnalysisThreadType", "CodeAnalysisIssue");
```

Comment:
Comment prettyfier ... the VSTS extension was done by the IDC team and it seems it's enabled for all accounts. Note that if the extension is disabled, PRCA will still work, but withouth the pretty pictures